conf/pf.conf.defaults

```diff
@@ -1364,7 +1364,10 @@ fingerbank_data_update_interval=6h
 #
 # Interval at which PacketFence updates the local Fingerbank data from the cloud API
 cluster_check_interval=1m
-
+# maintenance.option_82_task_interval
+#
+# Interval at which PacketFence will query compatibles switches for option 82 switch mac address
+option_82_task_interval=23h
```

Comment:
s/option_82_task_interval/option82_query_interval ?

IMO, "task" is a bit ... generic...